mantidproject/mantid
# Remove rst files for deleted algorithms

**Description of work**
Remove rst files for deleted algorithms.
Ignoring `DSFInterp` as that is handled by #xxxxx

**To test:**
(1) Build docs
(2) Run script by @AndreiSavici 
```
from mantid.simpleapi import *
import glob 

logger.error('algorithms')

files=[part.split('/')[-1].replace('.html','') for part in glob.glob('/home/3y9/Mantid/Build/docs/qthelp/algorithms/*.html')]
algs=mantid.api.AlgorithmFactory.getRegisteredAlgorithms(True)

for algname in files:
   if algname=='index':
       continue
   if '-v' in algname:
       algname, version = algname.split('-v')
       try:
           if not int(version) in algs[algname]:
               raise ValueError(algname+version)
       except:
           print(algname+version)
   else:
       if algname not in algs.keys():
           print(algname)


logger.error('fit functions')
files=[part.split('/')[-1].replace('.html','') for part in glob.glob('/home/3y9/Mantid/Build/docs/qthelp/fitting/fitfunctions/*.html')]
fitfunc=mantid.api.FunctionFactory.getFunctionNames()

for f in files:   
    if f=='index':
       continue
    if f not in fitfunc:
        print(f)
        
```
(3) Check no .rst file for unregistered algorithm



*There is no associated issue.*





---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
